### PR TITLE
Fix facility datatype.

### PR DESCRIPTION
**Pull request by mgsergio**
@VladiMihaylenko @mpimenov

### CODE REVIEW

**Comment by mpimenov** [comment]
LGTM

File: map/booking_api.cpp [hunk 1]
```diff
@@ -150,7 +150,11 @@ void BookingApi::GetHotelInfo(string const & hotelId, string const & /* lang */,
       "http://storage8.static.itmages.ru/i/16/0915/h_1473944905_4312757_433c687f4d.jpg",
       "http://storage6.static.itmages.ru/i/16/0915/h_1473945191_1817571_b945aa1f3e.jpg"});
 
-  info.m_facilities = {"Non smoking rooms", "Gym", "Pets are allowed"};
+  info.m_facilities = {
+    {"no_smoking_rooms", "Non smoking rooms"},
```
Comment (mpimenov) [inline map/booking_api.cpp]: s/no_/non_/

File: map/booking_api.cpp [hunk 2]
```diff
@@ -150,7 +150,11 @@ void BookingApi::GetHotelInfo(string const & hotelId, string const & /* lang */,
...
+    {"gym", "Training gym"},
+    {"pets_are_allowed", "Pets are allowed"}
```
Comment (mpimenov) [inline map/booking_api.cpp]: pets_allowed как вариант

File: map/booking_api.cpp (hunk 1 above)
Comment (mgsergio) [inline map/booking_api.cpp]: Done.

**Pull request merged**